private/private.h

```diff
@@ -28,7 +28,12 @@
 #define __DISPATCH_PRIVATE__
 
 #ifdef __APPLE__
+#include <Availability.h>
+#include <os/availability.h>
 #include <TargetConditionals.h>
+#include <os/base.h>
+#else
```

Comment:
shouldn't this be `#elif defined(__linux__)` like in dispatch/dispatch.h, these headers are included by other platforms as well...